### DISCUSSION TITLE
Activity Log IP Adress disable

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#19489] [Feature Requests]
### Summary

I would like to use directus as a backend. But the activity log saves also the IP-adress of a user. Also having the mail address i could identify a user.
This stands against EU-laws i think? https://iapp.org/news/a/are-ip-addresses-generated-when-users-visit-websites-personal-information/#:~:text=Under%20the%20EU%20General%20Data,header%20information%20that%20website%20hosts

Could someone help me in this topic? 
Can i disable the ip adress saving somehow even for System logins?

### Motivation

Legal Requirements

### Detailed Design

Allow the disable & track activity for system collections?
Or a global setting to completly not save the ip adress?

### Must have:
Complain with EU law

### Drawbacks

Tradeoff would be to completly disable activity log

### Alternatives

Switching to another CMS i guess?

### Adoption Strategy

Writing a hook to save or discard the IP Adress?

**Comment by NilsBaumgartner1994** [comment]
Another idea would be to delete the activity logs after X days.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
